rest/snapshot.go

```diff
@@ -17,7 +17,7 @@ const (
 	ListOptionsChainSnapshotPath  = "/v3/snapshot/options/{underlyingAsset}"
 	GetCryptoFullBookSnapshotPath = "/v2/snapshot/locale/global/markets/crypto/tickers/{ticker}/book"
 	GetIndicesSnapshotPath        = "/v3/snapshot/indices"
-	ListAssetSnapshots            = "/v3/snapshot"
+	ListAssetSnapshotsPath        = "/v3/snapshot"
```

Comment:
This is a breaking change. Its an exported field that you are removing. Might not matter as much since I haven't merged [my pr](https://github.com/polygon-io/client-go/pull/286/files#diff-27688c042992efb8fb7f4c17257e043c3301e235060ed5ff018437cd4b45dbaaR23) yet which introduces a new name.

Reply:
Great catch, thanks!

This is updated to the latest from master (your change in [this pr](https://github.com/polygon-io/client-go/pull/286/files)).